### DISCUSSION TITLE
Make reordering cards possible

### DISCUSSION THREAD

**Discussion by ThomasMahon** [GitbookIO/community#63] [Feature Requests]
### What's your feature/idea?

Cards cant be swapped around whereas their derivative control - tables - allow their rows and columns to be adjusted.

Similarly the contents of the card cant be changed, which is a major problem if you add a new column (property) to a card and want to swap its location with another property. There is no way to do it.

### What's your desired outcome?

Ability to change order of cards by drag-move. 
Ability to change the order of columns (properties) in cards.

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by volkantash** [comment]
Turn the cards into tables and reorder them until this problem is resolved. ;)

**Reply by ThomasMahon** [reply]
@volkantash that was one of the first things I tried, but switching between cards and tables resets the order so it does nothing - how have you got it to work? I'm using Chrome so is this maybe a bug in the Browser or did you just speak too soon?

**Reply by ThomasMahon** [reply]
@volkantash Ignore my reply I've got mixed up between reordering cards and my other feature request to re-order fields in cards! Thanks for the reply.

**Reply by volkantash** [reply]
> @volkantash Ignore my reply I've got mixed up between reordering cards and my other feature request to re-order fields in cards! Thanks for the reply.

**My way**

Create card
Create fields
If need ordering
Click option/drag button
Select table style
Use table's drag buttons
Click option/drag button
Select card style
Check other options.
Done!

> _"Yes. I don't like doing that either. However, until this uselessness is over, I am content with this method."_

**Reply by yevvie** [reply]
> Turn the cards into tables and reorder them until this problem is resolved. ;)

I've just noticed that reordering them currently does not persist. Tried multiple times and the only way to keep the table fields reordered is to create a new table with the same content.

**Comment by taszap** [comment]
Hey everyone thanks so much for your feedback - we agree that using cards can be tricky and are working on fixes and improvements. We will update you in due course

**Comment by taszap** [comment]
Just a quick update that this feature has been shipped. 
You should be able to reorder cards but if you encounter issues please don't hesitate to reach out to us on support@gitbook.com
